lib/edit-foods.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(function() {
+  $('tbody').on('click', '.food-row td', function() { 
+    var $td = $(this);
+    var dataName = $td.attr('class');
+    var currentFoods = JSON.parse(localStorage.getItem( "food-items" ));
+    var food = findEditedFood(currentFoods, $td.text());
+    currentFoods = pullOutEditedFood(currentFoods, food);
+    var $input = replaceTdWithInput($td); 
+    
+    var save = function() {
+      updateFood(food, dataName, $input.val());
+      currentFoods.push(food);
+      localStorage.setItem('food-items', JSON.stringify(currentFoods)); 
+      $input.parent().remove();
+      formatFoodItemTable(food.name, food.calories);
+    };
+    
+    $input.one('blur', save).focus();
+  });
+
+});
+
+function updateFood(food, dataName, newData){
+  if (dataName === 'food-name') {  
+    food.name = newData 
+  } else {
+    food.calories = newData
+  }
+};
+
```

Comment:
Love how we were able to get the update calories and food name into the single method!